easybuild/scripts/rpath_args.py

```diff
@@ -93,6 +93,8 @@
 
     # filter out --enable-new-dtags if it's used;
```

Comment:
@jdonners please also fix this comment, to something like `# replace --enable-new-dtags with --disable-new-dtags if it's used`

the last bit makes me wonder: should we just *always* inject `--disable-new-dtags` (maybe combined with filtering out `--enable-new-dtags`, to be sure)?